Depend-on/Bash.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+identify_and_run() {
+    os=$(uname)
+    if [ "$os" = "Linux" ]; then
+        distro=$(lsb_release -si)
+        case "$distro" in
+            CentOS|RedHatEnterpriseServer)
```

Comment:
What about:
* Rocky
* Oracle
* Fedora
* Scientific Linux

I'm sure I'm missing others.

Reply:
thats just for the moment when I become more familiar with them I'll add them

(im only 12)